src/main/java/com/epam/javacourses2016/task8/SolverTask8.java

```diff
@@ -16,6 +18,22 @@
      * @return true - скобки расставлены верно, иначе - false.
      */
     public boolean isNormalBrackets(String string) {
-        return false;
+        Stack<Character> characters = new Stack<>();
```

Comment:
Stack is legacy, use Deque instead
`Deque<Integer> stack = new ArrayDeque<Integer>();`